example/basic/index.js

```diff
@@ -6,6 +6,13 @@ const app = new Hono()
 // Mount Builtin Middleware
 app.use('*', Middleware.poweredBy())
 app.use('*', Middleware.logger())
+app.use(
+  '*',
+  Middleware.basicAuth({
```

Comment:
I think it's good to use more general words about `name` and `pass`.
Such as *username*/*password*.